browser/src/control/Toolbar.js

```diff
@@ -451,6 +451,7 @@ L.Map.include({
 			else if (map.getDocType() === 'drawing') {
 				document.getElementById('drawing-shortcuts').style.display='block';
 			}
+			document.getElementById('online-help-search-input').style.display='none';
```

Comment:
I have disabled the filter content feature in keyboard shortcut dialog. 

Note: All the HTML changes are done for  keyboard-shortcut dialog, so if we need to this feature just remove this line and all works good 😉 
 
CC: @pedropintosilva

Reply:
Yes, that would be great. But let's not do it in this PR. First we need to make sure the new feature is in for the Help dialog and then we can remove this line (enable it for keyboard shortcuts dialog) and re-test it to make sure everything is working properly

Reply:
okay 👍🏼